src/System.Private.ServiceModel/src/System/ServiceModel/Dispatcher/DataContractSerializerOperationFormatter.cs

```diff
@@ -35,10 +35,11 @@ internal static DataContractSerializer CreateSerializer(Type type, string rootNa
 
         internal static DataContractSerializer CreateSerializer(Type type, IList<Type> knownTypes, string rootName, string rootNs, int maxItems)
         {
+            XmlDictionary dictionary = new XmlDictionary(2);
             return new DataContractSerializer(
                 type,
-                rootName,
-                rootNs,
+                dictionary.Add(rootName),
+                dictionary.Add(rootNs),
```

Comment:
@mconnew  can you please open an issue in corefx against DCS? No need to change code here.